test/parallel/test-repl.js

```diff
@@ -180,6 +180,13 @@ function error_test() {
       expect: prompt_multiline },
     { client: client_unix, send: '})()',
       expect: '1' },
+    // Multiline function call
+    { client: client_unix, send: 'function f(){}; f(f(1,',
+      expect: prompt_multiline },
+    { client: client_unix, send: '2)',
+      expect: prompt_multiline },
+    { client: client_unix, send: ')',
+      expect: 'undefined' },
```

Comment:
@Zirak `prompt_multiline` is `...` and `prompt_unix` is the `>` which we get in the REPL.